tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -3,24 +3,31 @@
 from pathlib import Path
 import copy
 
+# lib
+import confuse
+
 # project
 from src.chia_log.handlers.wallet_added_coin_handler import WalletAddedCoinHandler
 from src.notifier import EventType, EventService, EventPriority
-from src.config import Config
 
 
 class TestWalledAddedCoinHandler(unittest.TestCase):
     def setUp(self) -> None:
         config_dir = Path(__file__).resolve().parents[3]
-        config = Config(config_dir / "config-example.yaml")
-        self.handler_config = config.get_handlers_config()[WalletAddedCoinHandler.config_name()]
+        self.config = confuse.Configuration("chiadog", __name__)
+        self.config.set_file(config_dir / "src/default_config.yaml")
+        self.handler_config = self.config["handlers"][WalletAddedCoinHandler.config_name()]
 
-        self.handler = WalletAddedCoinHandler(config=None)
+        self.handler = WalletAddedCoinHandler(config=self.handler_config)
         self.example_logs_path = Path(__file__).resolve().parents[1] / "logs/wallet_added_coin"
 
+    def tearDown(self) -> None:
+        # self.handler_config.clear()
```

Comment:
Maybe you wanted to either delete or uncomment?

Reply:
Yup, good catch. SubViews can't be cleared so it was useless.